core/src/test/java/io/kestra/core/tasks/test/PollingTrigger.java

```diff
@@ -0,0 +1,42 @@
+package io.kestra.core.tasks.test;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.conditions.ConditionContext;
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.models.triggers.AbstractTrigger;
+import io.kestra.core.models.triggers.PollingTriggerInterface;
+import io.kestra.core.models.triggers.TriggerContext;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import jakarta.validation.constraints.NotNull;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+
+@SuperBuilder
+@NoArgsConstructor
+public class PollingTrigger extends AbstractTrigger implements PollingTriggerInterface {
+    @PluginProperty
+    @NotNull
+    @Builder.Default
+    private Long duration = 2000L;
```

Comment:
```suggestion
    private Long duration = 1000L;
```
Please avoid too long wait time in test to make them quicker